multirun/def.bzl

```diff
@@ -25,6 +25,11 @@ def _multirun_impl(ctx):
     runnerInfo = ctx.attr._runner[DefaultInfo]
     runfiles = runfiles.merge(runnerInfo.default_runfiles)
 
+    for data_dep in ctx.attr.data:
+        default_runfiles = data_dep[DefaultInfo].default_runfiles
```

Comment:
I have not actually written any rules, but this seems to be getting the dep's runfiles, when don't you want the dep *as a runfile*? I think this will get the transitive deps but not the direct dep.

Reply:
Maybe the right fix is something like:

```python
for data_dep in ctx.attr.data:
  runfiles = runfiles.merge(ctx.runfiles(files = data_dep))
```

Reply:
Yeah, this is what was not clear to me, the docs are vague. I'll update the PR.